### PR TITLE
Check all files in `src/test` for `borrowck_graphviz_postflow`

### PR DESCRIPTION
**Pull request by ecstatic-morse**
This attribute causes DOT files to be generated in the top-level directory. It is intended to be used only temporarily and should never appear on master. This also tells git to ignore DOT files in the root or the  `mir_dump` directory, which `-Z dump-mir` uses by default.

This will prevent #65071 from occurring again. It needs to be merged after #65629, otherwise `tidy` will start failing.

r? @Mark-Simulacrum

### CODE REVIEW

**Comment by ecstatic-morse** [comment]
This should fail `tidy`.

**Review by Mark-Simulacrum — APPROVED** [review]
r=me when the other PR is merged

File: .gitignore [hunk 1]
```diff
@@ -52,3 +52,6 @@ config.stamp
 Session.vim
 .cargo
 no_llvm_build
+# Generated when dumping Graphviz output for debugging:
+/mir_dump/
+/*.dot
```
Comment (RalfJung) [inline .gitignore]: I think this should be reverted. In fact I just removed these not too long ago.^^

I do sometimes create `mir_dump` folders during debugging, and those are junk that I want to clean up. They should *not* be automatically ignored by git. If you are okay with junk in your working dir, that's okay -- git has `.git/info/exclude` for you to locally configure stuff you want ignored. But please don't interfere with other people who want to keep their working dir clean.

Comment (RalfJung) [reply .gitignore]: In particular see the comment at the top of this file:
```
# This file should only ignore things that are generated during a build,
# generated by common IDEs, and optional files controlled by the user
# that affect the build (such as config.toml).
```

**Pull request merged**